data-prepper-plugins/mutate-event-processors/README.md

```diff
@@ -56,7 +56,8 @@ then when we run with the same input, the processor will parse the message into
 
 ### Configuration
 * `entries` - (required) - A list of entries to add to an event
-  * `key` - (required) - The key of the new entry to be added
+  * `key` - (required) - The key of the new entry to be added. One of `key` or `metadata_key` is required.
+  * `metadata_key` - (required) - The key of the new metadata attribute to be added. One of `key` or `metadata_key` is required.
```

Comment:
This doesn't support json pointer right? Should we mention that in the documentation here?

Reply:
There are mutually exclusive in the code and it appears `key` takes presedence over `metadata`.

Reply:
It does not support Json pointer. I have updated the documentation.